### PR TITLE
Update PowerShell to 7.1.0-preview.2

### PR DESCRIPTION
**Pull request by adityapatwardhan**
Update the PowerShell version to 7.1.0-preview.2. This is based on .NET 5 Preview 3.

### CODE REVIEW

File: 5.0/sdk/nanoserver-1909/amd64/Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ RUN `
     Remove-Item -Force dotnet.zip; `
     `
     # Install PowerShell global tool
-    $powershell_version = '7.1.0-preview.1'; `
+    $powershell_version = '7.1.0-preview.2'; `
     Invoke-WebRequest -OutFile PowerShell.Windows.x64.$powershell_version.nupkg https://pwshtool.blob.core.windows.net/tool/$powershell_version/PowerShell.Windows.x64.$powershell_version.nupkg; `
     $powershell_sha512 = '8ba8c4226ffc5120782335c46a747c862141b86c3e8ef8833936d2d3fafd544bcc6f4590723caae3e631a23d16748455ceb526799aba50e090a73750363792ba'; `
```
Comment (MichaelSimons) [inline 5.0/sdk/nanoserver-1909/amd64/Dockerfile:26]:
```suggestion
    $powershell_sha512 = '1f030b2c25070a208bbbdf911a31befcfec54f69b783cd77419f612290bdd0678ddfed96df225e762dbf126adc65bf86be9385176b705e1379a0369fb11b0f97'; `
```

**Comment by mthalman** [comment]
This is marked as WIP.  Is there work remaining for this?

**Comment by adityapatwardhan** [comment]
Just going through final validations for the release. Need about 1 hour. I will remove WIP after that.

**Comment by adityapatwardhan** [comment]
@mthalman This is ready for merge now

**Pull request merged**